models/classes/class.QtiTestService.php

```diff
@@ -1296,4 +1312,20 @@ private function getQtiPackageImportPreprocessing()
     {
         return $this->getServiceLocator()->get(QtiPackageImportPreprocessing::SERVICE_ID);
     }
+
+    /**
+     * @return \oat\oatbox\service\ConfigurableService|TreeService
+     */
+    private function getItemTreeService()
```

Comment:
We do not care it is an instance of `ConfigurableService`, it is just there because legacy registry approach.

```suggestion
    private function getItemTreeService(): TreeService
```

Reply:
fixed